### PR TITLE
Feat: added alert on product page added to wishlist/cart

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
### Add alert
- alert on add to cart 
- alert on add to wishlist 
on product page only

https://user-images.githubusercontent.com/60725346/161414042-743f80ca-04f2-4a6b-931c-d10265612469.mov

### CODE REVIEW

File: src/components/alert/alert.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const Alert = ({ text }) => {
+  return (
+    <div className="alert alert-secondary">
+      <i className="bi bi-chevron-double-right"></i>{" "}
```
Comment (sakshi006) [inline src/components/alert/alert.js:4]: You can remove these quotes

**Review by sakshi006 — APPROVED** [review]
the code is clean and i liked the logic for alerts

**Review by teja538 — APPROVED** [review]
code looks clean 
good UI

File: src/pages/homePage/home.css [hunk 2]
```diff
@@ -60,7 +60,15 @@
 .gift-section {
   margin: 2rem 0.4rem;
 }
-
+.alert {
+  position: fixed;
+  right: 40px;
+  top: 120px;
+  z-index: 111;
```
Comment (sashaboi) [inline src/pages/homePage/home.css:67]: :) z index 111 🥇

Comment (harshitpaliwal95) [reply src/pages/homePage/home.css:67]: under rated value 🙃

**Pull request merged**